<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datacorn</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/res.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTabWidget" name="dbEditorTabsWidget">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuDatabase">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionNewDatabase"/>
    <addaction name="actionOpenDatabase"/>
    <addaction name="actionShowClipboardDatabase"/>
    <addaction name="actionCloseDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHomepage"/>
   </widget>
   <widget class="QMenu" name="menuCard">
    <property name="title">
     <string>&amp;Card</string>
    </property>
    <addaction name="actionNewCard"/>
    <addaction name="actionSaveData"/>
    <addaction name="actionDeleteData"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopySelectedCards"/>
    <addaction name="actionPasteClipboardCards"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuCard"/>
   <addaction name="menuAction"/>
   <addaction name="menuHelp"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionNewDatabase">
   <property name="text">
    <string>&amp;New Database...</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>&amp;Open Database...</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>&amp;Homepage</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionCloseDatabase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close Database</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNewCard">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;New Card...</string>
   </property>
  </action>
  <action name="actionDeleteData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Data</string>
   </property>
  </action>
  <action name="actionCopySelectedCards">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Copy Selected Cards</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionPasteClipboardCards">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Paste Clipboard Cards</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="actionShowClipboardDatabase">
   <property name="text">
    <string>&amp;Show Clipboard Database</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
